.github/workflows/Android-CI.yml

```diff
@@ -0,0 +1,33 @@
+name: PullRequest
+
+on: [pull_request]
+
+jobs:
+  buildTest:
+    name: Build SDK and HelloCardboard
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.2
+        with:
+          fetch-depth: 0
+      - name: Build sdk
+        run: ./gradlew :sdk:assemble
+      - name: Build project
+        run: ./gradlew :hellocardboard-android:assembleDebug
+  Check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.2
+        with:
+          fetch-depth: 0
+      - name: Code checks
```

Comment:
Line 26 and 27 should move before line 17. We don't want to checkout the project again.